services/Makefile

```diff
@@ -14,6 +14,9 @@ lint: depends
 unit:
 	$(COMPOSE) run -e NODE_ENV=test --rm node npm run jest
 
+watch:
+	$(COMPOSE) run -e NODE_ENV=test --rm node jest --watchAll
```

Comment:
IIUC reading docs, this is an option to constantly run test while one is developing (?)

Reply:
Yes, this is what most Ruby/Python/JavaScript tools support under the name `watch`